src/openforms/appointments/base.py

```diff
@@ -220,3 +225,18 @@ def get_appointment_details_html(self, identifier: str) -> str:
         return render_to_string(
             "appointments/appointment_details.html", {"appointment": details}
         )
+
+    def get_appointment_links(self, submission: Submission) -> Dict[str, str]:
+
+        token = submission_appointment_token_generator.make_token(submission)
+
+        cancel_path = reverse(
+            "appointments:appointments-verify-cancel-appointment-link",
+            kwargs={
+                "token": token,
+                "submission_uuid": submission.uuid,
+            },
+        )
+        cancel_url = f"{settings.BASE_URL}{cancel_path}"
```

Comment:
Is this the best way to build the full url?  We don't have access to the request so we can't do `request.build_absolute_url()`(or whatever the exact function is) and even if we did I'm not sure that would be the best way to do it?

Reply:
Yeah I think it is, for now at least. It does mean that we use the `BASE_URL` setting for more than DigiD, so it should be documented as such.

An alternative is using `django.contrib.sites` but I really dislike that and brings its own problems, so let's improve on documentation here.

However, don't join URLs together like that, as the `BASE_URL` setting may have a trailing slash. To be safe, you should use `urllib.parse.urljoin`.